js/data.js

```diff
@@ -0,0 +1,64 @@
+import {getRandomInteger} from './utils.js';
+import {getRandomElement} from './utils.js';
```

Comment:
Экспортировать из одного модуля надо одной записью
```suggestion
import {getRandomInteger, getRandomElement} from './utils.js';
```